search.go

```diff
@@ -67,13 +61,15 @@ func (dr *dateTimeRange) ParseDates(dateTimeParser analysis.DateTimeParser) (sta
 			start = s
 		}
 	}
+
```

Comment:
Nit: Drop this and all the other empty lines introduced in the PR.

Reply:
I changed it as you said.

Reply:
> I changed it as you said.

I changed it as you said.

Reply:
> Nit: Drop this and all the other empty lines introduced in the PR.

I changed it as you said.